src/main/java/core/mate/academy/service/MachineServiceImpl.java

```diff
@@ -1,7 +1,41 @@
 package core.mate.academy.service;
 
+import core.mate.academy.model.Machine;
+import java.util.Collections;
+import java.util.List;
+
 /**
  * Your implementation of MachineService.
  */
-public class MachineServiceImpl {
+public class MachineServiceImpl implements MachineService<Machine> {
+    @Override
+    public List<Machine> getAll(Class type) {
+        MachineProducer machineProducer;
+        switch (type.getSimpleName()) {
+            case "Bulldozer":
```

Comment:
try this. if it will not work, just rewrite with `if (type.equals(Bulldozer.class)) ...`, because using String literals here is bad practice
```suggestion
        switch (type) {
            case Bulldozer.class:
```